app/src/main/java/fr/free/nrw/commons/wikidata/WikidataEditService.java

```diff
@@ -64,6 +64,11 @@ public void createClaimWithLogging(String wikidataEntityId, String fileName) {
             return;
         }
 
+        if (!p18Value.trim().isEmpty()) {
+            Timber.d("Skipping creation of claim as p18Value is not null, we won't override existing image");
```

Comment:
This log is lying, this isn't "not null" it is "not blank"(to borrow the expression from the kotlin standard library), if this method accepts null values for the p18 then this is a guaranteed NPE

Reply:
Yeah, fixed:) it is never null, empty or value

Reply:
Excellent, please mark the parameter `@NonNull`

Reply:
Done @macgills thanks a lot:)